### DISCUSSION TITLE
Sort Results Via API??

### DISCUSSION THREAD

**Discussion by Dark-Akkula** [supabase/supabase#20248] [Questions] [answered]
Can someone help me to figure out how to sort data coming from the api?  I believe it is possible but I am not sure how it works in the API docs.

**Comment by GaryAustin1** [comment]
Are you using something like the supabase-js client?
https://supabase.com/docs/reference/javascript/order

**Reply by Dark-Akkula** [reply]
Hey, I am actually trying to get the data via the API call from flutterflow so I am trying to see if I can do that same operation through the API call.

**Reply by GaryAustin1** [reply, marked as answer]
I'm guessing you mean the REST API directly....
https://postgrest.org/en/stable/references/api/tables_views.html?highlight=order#ordering
Those docs are what you should go to for the most detail.  It covers headers for things like limits as well as url formats.

**Reply by Dark-Akkula** [reply]
Thanks, that is very helpful!
